docsrc/Makefile

```diff
@@ -0,0 +1,31 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
+SPHINXPROJ    = pysal
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+github:
+	@make html
+	@cp -a _build/html/. ../docs
+
+
+
+clean:
+	rm -rf $(BUILDDIR)/*
+	rm -rf auto_examples/
+	rm -rf ../docs/*
```

Comment:
Should this line be removed?

Reply:
It cleans out possibly stale static files before a build.
Maybe this should be replaced with an rsync and deletion flag

Reply:
So `make clean` before `make github`? 
If that is the case, might make sense to append a cleaning line at the end of  `github`:
`github:
	@make html
	@cp -a _build/html/. ../docs
        @rm -rf $(BUILDDIR)/*
`
to avoid pushing duplicating built html files (in docsrc/(BUILDDIR)/* and /docs) to github by `git add .` .